Warehouse.Infrastructure/RestClient/Configuration/MockyProductsConfiguration.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+namespace Warehouse.Infrastructure.RestClient.Configuration
+{
+	public class MockyProductsConfiguration
+    {
+		public required string Url { get; set; }
```

Comment:
Use this for your configuration of the approach
 and adjust these props here
```
 "MockyClient": {
    "BaseUrl": "http://www.mocky.io/",
    "ProductUrl": "v2/5e307edf3200005d00858b49"
  }
```

Reply:
Done.